examples/fabric-admin/scripts/fabric-sync-app.py

```diff
@@ -0,0 +1,320 @@
+    await p.run()
+    return p
+
+
+async def main(args):
+
+    # Node ID of the bridge on the fabric.
+    bridge_node_id = 1
+
+    if args.commissioner_node_id == bridge_node_id:
+        raise ValueError(f"NodeID={bridge_node_id} is reserved for the local fabric-bridge")
+
+    storage_dir = args.storage_dir
+    if storage_dir is not None:
+        os.makedirs(storage_dir, exist_ok=True)
+    else:
+        storage = TemporaryDirectory(prefix="fabric-sync-app")
+        storage_dir = storage.name
+
+    pipe = args.stdin_pipe
+    if pipe and not os.path.exists(pipe):
+        os.mkfifo(pipe)
+
+    def terminate(signum, frame):
+        admin.terminate()
+        bridge.terminate()
+        sys.exit(0)
+
+    signal.signal(signal.SIGINT, terminate)
+    signal.signal(signal.SIGTERM, terminate)
+
+    admin, bridge = await asyncio.gather(
+        run_admin(
+            args.app_admin,
+            storage_dir=storage_dir,
+            rpc_admin_port=args.app_admin_rpc_port,
+            rpc_bridge_port=args.app_bridge_rpc_port,
+            paa_trust_store_path=args.paa_trust_store_path,
+            commissioner_name=args.commissioner_name,
+            commissioner_node_id=args.commissioner_node_id,
+            commissioner_vendor_id=args.commissioner_vendor_id,
+        ),
+        run_bridge(
+            args.app_bridge,
+            storage_dir=storage_dir,
+            rpc_admin_port=args.app_admin_rpc_port,
+            rpc_bridge_port=args.app_bridge_rpc_port,
+            secured_device_port=args.secured_device_port,
+            discriminator=args.discriminator,
+            passcode=args.passcode,
+        ))
+
+    # Wait a bit for apps to start.
+    await asyncio.sleep(1)
+
+    try:
+        # Check whether the bridge is already commissioned. If it is,
+        # we will get the response, otherwise we will hit timeout.
+        await admin.send(
+            f"descriptor read device-type-list {bridge_node_id} 1 --timeout 1",
+            # Log message which should appear in the fabric-admin output if
+            # the bridge is already commissioned.
+            expected_output="Reading attribute: Cluster=0x0000_001D Endpoint=0x1 AttributeId=0x0000_0000",
+            timeout=1.5)
+    except asyncio.TimeoutError:
+        # Commission the bridge to the admin.
+        cmd = f"fabricsync add-local-bridge {bridge_node_id}"
+        if args.passcode is not None:
+            cmd += f" --setup-pin-code {args.passcode}"
+        if args.secured_device_port is not None:
+            cmd += f" --local-port {args.secured_device_port}"
+        await admin.send(
+            cmd,
+            # Wait for the log message indicating that the bridge has been
+            # added to the fabric.
+            f"Commissioning complete for node ID {bridge_node_id:#018x}: success")
+
+    # Open commissioning window with original setup code for the bridge,
+    # so it can be added by the TH fabric.
```

Comment:
supernit, just say that so the application is commissionable. I don't think we need to mention TH fabric here since technically this can be used outside of tests